sphinx_doc/howtos/tutorials/docs/navigation2_with_turtlebot3_in_gazebo.rst

```diff
@@ -0,0 +1,195 @@
+.. _navigation2-with-turtlebot3-in-gazebo:
+
+Navigation2 with Turtlebot 3 in Gazebo
+**************************************
+
+- `Overview`_
+- `Requirements`_
+- `Tutorial Steps`_
+- `Videos`_
+
+Overview
+========
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 in Gazebo.
+Turtlebot is a low-cost, personal robot kit with open-source software (ROS2).
+Turtblebot robots are widely supported by the ROS community.
+You can find many applications and examples of Turtlebot projects on the Internet.
+You can find more information about Turtlebot3 `here. <http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/>`_
+
+Links to the robot kits and Github repo.
+
+- `Turtlebot Burger <http://www.robotis.us/turtlebot-3-burger-us/>`_
```

Comment:
Please point them to this link: https://www.turtlebot.com/

This is the official turtlebot community website, which contains these links and other helpful information. Also Robotis isn't the only vendor (see examples https://www.turtlebot.com/purchase/) so I don't want to be "advertising" for a particular single vendor. If you're not in Korea or US, robotis isn't probably the best option.

Reply:
Isn't TB3 a Robotis robot? If yes, I think it is OK to use the original manufacturer's web site for the product info.

Reply:
No, TB3 is a standard that uses robotis parts.